native/src/routes/search/containers/SearchModalContainer.js

```diff
@@ -59,22 +59,24 @@ const mapStateToProps = (state: StateType, ownProps: OwnPropsType) => {
 
 const mapDispatchToProps = (dispatch: Dispatch<StoreActionType>) => ({ dispatch })
 
-const ThemedTranslatedSearch = withTheme(
-  withTranslation('search')(SearchModal)
+const ThemedTranslatedSearch = withTheme<$Diff<SearchModalPropsType, {| t: TFunction |}>>(
+  withTranslation<SearchModalPropsType>('search')(SearchModal)
 )
 
 const SearchModalContainer = (props: PropsType) => {
-  const { dispatch, navigation, ...rest } = props
+  const { dispatch, navigation, route, cityCode, ...rest } = props
 
   const navigateToLinkProp = useCallback((url: string, language: string, shareUrl: string) => {
     const navigateTo = createNavigate(dispatch, navigation)
     navigateToLink(url, navigation, language, navigateTo, shareUrl)
   }, [dispatch, navigation])
 
-  return <ThemedTranslatedSearch
-    {...rest}
-    navigateToLink={navigateToLinkProp}
-    navigateTo={createNavigate(dispatch, navigation)} />
+  return cityCode
+    ? <ThemedTranslatedSearch cityCode={cityCode}
+                              navigateToLink={navigateToLinkProp}
+                              navigateTo={createNavigate(dispatch, navigation)}
+                              {...rest} />
+    : null
```

Comment:
Does this case actually occur?

Reply:
I don't think so, no. Necessary for flow though.